easybuild/easyconfigs/p/pkgconfig/pkgconfig-1.5.1-GCCcore-8.2.0-python.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'PythonBundle'
+
+name = 'pkgconfig'
+version = '1.5.1'
+versionsuffix = '-python'
+
+homepage = 'http://github.com/matze/pkgconfig'
+description = """pkgconfig is a Python module to interface with the pkg-config command line tool"""
+
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+
+multi_deps = {'Python': ['3.7.2', '2.7.15']}
+
+builddependencies = [('binutils', '2.31.1')]
+
+dependencies = [('pkg-config', '0.29.2')]
+
+use_pip = False
+
+exts_list = [
+    ('poetry', '0.12.12', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/poetry/'],
+        'checksums': ['726420e7519e069d9245ebac3f1c367b7ef9aca420c6911a9261772c7c814a2d'],
+    }),
+    (name, version, {
+        'source_urls': ['https://pypi.python.org/packages/source/p/pkgconfig/'],
+        'checksums': ['97bfe3d981bab675d5ea3ef259045d7919c93897db7d3b59d4e8593cba8d354f'],
+    }),
+]
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages/'],
```

Comment:
@akesandgren You can drop this, it's the default with `PythonBundle` (and there's nothing else to check for in this case)